<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevelGUIWindow</class>
 <widget class="QMainWindow" name="DevelGUIWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1330</width>
    <height>949</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Development GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="tbPlayPause">
        <property name="toolTip">
         <string>Play</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/res/play.svg</normaloff>:/icons/res/play.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbStop">
        <property name="toolTip">
         <string>Stop</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/res/stop.svg</normaloff>:/icons/res/stop.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbRepeat">
        <property name="toolTip">
         <string>Repeat</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/res/repeat.svg</normaloff>:/icons/res/repeat.svg</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>FPS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbFPS">
        <property name="toolTip">
         <string>Max FPS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbRefresh">
        <property name="toolTip">
         <string>Recalculate Frame</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/res/refresh.svg</normaloff>:/icons/res/refresh.svg</iconset>
        </property>
        <property name="shortcut">
         <string>F5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbLeft">
        <property name="toolTip">
         <string>Last Frame</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/res/left.svg</normaloff>:/icons/res/left.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbFrame">
        <property name="toolTip">
         <string>Frame Number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbRight">
        <property name="toolTip">
         <string>Next Frame</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/res/right.svg</normaloff>:/icons/res/right.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="hsFrame">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mainLayout">
      <item>
       <widget class="QGraphicsView" name="view"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Algorithms</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pbAddAlgo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="cobAlgorithms"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QScrollArea" name="scrollArea">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>286</width>
              <height>208</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QToolBox" name="toolBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="page_2">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>268</width>
                  <height>158</height>
                 </rect>
                </property>
                <attribute name="label">
                 <string>Page 2</string>
                </attribute>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Container</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="1" column="1">
                <widget class="QToolButton" name="tbContainerUp">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="arrowType">
                  <enum>Qt::UpArrow</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="6">
                <widget class="QListView" name="lvContainers"/>
               </item>
               <item row="1" column="3">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="2">
                <widget class="QToolButton" name="tbContainerDown">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="arrowType">
                  <enum>Qt::DownArrow</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="4">
                <widget class="QPushButton" name="pbAddContainer">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>60</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Add</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="5">
                <widget class="QPushButton" name="pbRemoveContainer">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>60</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Remove</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1" colspan="5">
                <widget class="QFrame" name="frameContainerContent">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QToolButton" name="tbContainerContentLeft">
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="arrowType">
                     <enum>Qt::LeftArrow</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="lContainerContent">
                    <property name="text">
                     <string>1/1</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="tbContainerContentRight">
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="arrowType">
                     <enum>Qt::RightArrow</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Runtime</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Pipeline FPS</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Pipeline</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="lPluginRuntime">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="lPipelineRuntime">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="lPipelineFPS">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Algo</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1330</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_project"/>
    <addaction name="actionSave_project"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_last_project"/>
    <addaction name="separator"/>
    <addaction name="actionSave_current_image"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionLoad_last_project">
   <property name="text">
    <string>Load last project</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionLoad_project">
   <property name="text">
    <string>Load project</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="text">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionSave_current_image">
   <property name="text">
    <string>Save current image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
